Source/Manager.swift

```diff
@@ -9,6 +9,10 @@
 import SwiftyJSON
 
 public class Manager: NSURLProtocol {
+    typealias queryArtifacts = [String: [String]]?
+    typealias IDArtifacts = [Int]?
```

Comment:
I'd suggest removing the optional from the typealias and only using it where appropriate (eg function params).  That will make your code cleaner when you know you're using non-nil versions of QueyArtifacts and IDArtifacts